helion/autotuner/base_search.py

```diff
@@ -155,17 +160,29 @@ def cleanup(self) -> None:
         self._precompile_args_path = None
         self._precompile_result_counter = count()
 
-    def _clone_args(self, args: Sequence[object]) -> Sequence[object]:
+    def _clone_args(
+        self, args: Sequence[object], all_tensors: bool = False
+    ) -> Sequence[object]:
+        if (
+            not hasattr(self, "_mutated_tensor_args")
```

Comment:
when do we have the case where `_mutated_tensor_args` doesn't exist on `self`? (since we already add the attribute above)